GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSceneObject.java

```diff
@@ -1219,6 +1219,13 @@ public final BoundingVolume expandBoundingVolume(
     public final BoundingVolume expandBoundingVolume(final Vector3f center, final float radius) {
         return expandBoundingVolume(center.x, center.y, center.z, radius);
     }
+    
+    /*
+     * Check if the object is culled in the frame or not
```

Comment:
May be we could use it where we are traversing though all the scene-objects in a scene.